### PR TITLE
RedisCacheEnableNonSSLPort check and test

### PR DESCRIPTION
**Pull request by avivgu**
By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: checkov/terraform/checks/resource/azure/RedisCacheEnableNonSSLPort.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+class RedisCacheEnableNonSSLPort(BaseResourceCheck):
+    def __init__(self):
+        name = "Ensure that only SSL are enabled for Cache for Redis"
+        id = "CKV_AZURE_91"
+        supported_resources = ['azurerm_redis_cache']
+        categories = [CheckCategories.NETWORKING]
+        super().__init__(name=name, id=id, categories=categories, supported_resources=supported_resources)
+
+    def scan_resource_conf(self, conf):
```
Comment (nimrodkor) [inline checkov/terraform/checks/resource/azure/RedisCacheEnableNonSSLPort.py:13]: @avivgu Seems like a `BaseResourceValueCheck` to me

Comment (avivgu) [reply checkov/terraform/checks/resource/azure/RedisCacheEnableNonSSLPort.py:13]: When the param does not exist it should pass, BaseResourceValueCheck doing that?

Comment (nimrodkor) [reply checkov/terraform/checks/resource/azure/RedisCacheEnableNonSSLPort.py:13]: @avivgu Look for the optional parameter `missing_block_result` in the constructor

Comment (avivgu) [reply checkov/terraform/checks/resource/azure/RedisCacheEnableNonSSLPort.py]: cool, done

**Review by nimrodkor — APPROVED** [review]

**Pull request merged**